releases/cockroachcloud-04062020.md

```diff
@@ -0,0 +1,26 @@
+---
+title: What&#39;s New in CockroachCloud
+toc: true
+summary: Additions and changes in CockroachCloud.
+build_for: [cockroachcloud]
+---
+
+## April 06, 2020
+
+In addition to various updates, this beta release includes the following major highlights:
+
+- Free trials of CockroachCloud are now available. [Contact us](https://www.cockroachlabs.com/contact-sales/) to request a trial code.
+- CockroachCloud now supports VPC peering for clusters running on GCP. [Contact us](https://support.cockroachlabs.com/hc/en-us) to set up a VPC peering-enabled CockroachCloud cluster.
+
+### Security updates
+
+CockroachCloud now requires a user to have a CockroachCloud account before accepting an invite to join an Organization.
+
+### General changes
+
+- The hardware options displayed while [creating a cluster](../v19.2/cockroachcloud-create-your-cluster.html#step-4-select-the-hardware-configuration) have been renamed as "Option 1" and "Option 2".
```

Comment:
Just confirming that 19.2 is the doc you want to link here. Or would a liquid version snippet make sense, since this is also linked on 20.1 docs?

Reply:
Yeah..19.2 for now, since we don't have any clusters on 20.1 yet. CC versioning is something that needs work, but it's on my to-do list.